models/storyboard.js

```diff
@@ -19,14 +19,22 @@ function createNav(items) {
     const nav = [];
 
     items.forEach((item, id) => {
-        const [mediaType] = Object.keys(item);
-        // fallback for now until every type has a title field to fill in. (or something for nav)
-        const tmpTitle = item[mediaType].title ? item[mediaType].title : `${id} ${mediaType}`;
-        const title = tmpTitle.replace(/<\/?[^>]+(>|$)/g, "");
+      var elKey = Object.keys(item)[0];
+      const root = item[elKey];
+      var writeToNav = true;
+      if (root && root.in_nav && root.in_nav.suppress) {
```

Comment:
can we not have this so nested? Why not just store a boolean right on in_nav?

Reply:
Done